include/irr/video/IGPUPipelineCache.h

```diff
@@ -13,6 +14,8 @@ class IGPUPipelineCache : public core::IReferenceCounted
 
 public:
 	virtual void merge(uint32_t _count, const IGPUPipelineCache** _srcCaches) = 0;
```

Comment:
does Vulkan have this? If not then only CPU cache should have.

Reply:
yea, it has